gui-tk/godley.tcl

```diff
@@ -305,12 +305,16 @@ proc setGodleyDisplay {} {
     foreach c [info commands godleyWindow*.displayStyle] {
         $c $preferences(godleyDisplayStyle)
     }
-    foreach c [info commands godleyIcon.table.multipleEquities] {
-        $c $preferences(multipleEquities)
-    }    
     redrawAllGodleyTables
 }
-	
+
+# sets each individual Godley table mutliple equity column preference
+proc toggleEquityColumns id {
+	global preferences
+	tk_messageBox -message "Close and reopen the Godley table to enable/disable this feature" -type ok -parent .$id.table
```

Comment:
This shouldn't be necessary if the change of state isn reflected in appearance.

Reply:
I've removed it, thanks :)